Automattic/wp-calypso
# Communicate quick start deprecation instead of upsell

#### Changes proposed in this Pull Request

* This PR introduces a new component to display a deprecation notice and links to resources instead of our current Quick Start upsells.
* This is a slightly different approach to the one in #58867, which removes these paths altogether. AFAICT, we have a handful of page views coming in each day to paths with either `/checkout/offer-support-session` or `/checkout/offer-quickstart-session` as a prefix, so this ensures that we are gracefully handling the deprecation and getting the user to the help we're currently offering.

Thoughts on the content and links we show (including ordering!) are most welcome!

#### Testing instructions

* Open this branch locally or view it via the live branch
* For each of the following paths, navigate to the relevant URL, and confirm that you see the new card without the "Hold on" header text:
  - `/checkout/offer-quickstart-session`
  - `/checkout/offer-quickstart-session/[siteSlug]`
  - `/checkout/offer-support-session`
  - `/checkout/offer-support-session/[siteSlug]
* Next, find the receipt ID for a purchase and the site slug for the related blog.
* Navigate to each of the following paths and verify that we show the content with the "Hold on" header content:
  - `/checkout/offer-quickstart-session/[receiptId]/[siteSlug]`
  - `/checkout/offer-support-session/[receiptId]/[siteSlug]`

#### Screenshots

##### Content with "Hold on" header

<img width="771" alt="Screenshot 2021-12-10 at 14 04 54" src="https://user-images.githubusercontent.com/3376401/145572974-5b972987-d054-4647-97ef-6ef2da32e75b.png">

##### Content without "Hold on" header
<img width="771" alt="Screenshot 2021-12-10 at 14 05 11" src="https://user-images.githubusercontent.com/3376401/145572992-8c783bfc-701b-44f2-a8aa-058b361f312b.png">


Related to #58867
